app/src/main/java/com/edu/movie/screen/trending/MoviesTrendingFragment.kt

```diff
@@ -0,0 +1,50 @@
+package com.edu.movie.screen.trending
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.edu.movie.R
+import com.edu.movie.utils.TrendingMoviesParams
+import kotlinx.android.synthetic.main.fragment_movies_trending.*
+
+class MoviesTrendingFragment : Fragment() {
+    private var param: TrendingMoviesParams? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param = it.getSerializable(ARG_PARAM) as TrendingMoviesParams?
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_movies_trending, container, false)
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        param?.let {
+            textView.text = param.toString()
+            Toast.makeText(context, param.toString(), Toast.LENGTH_SHORT).show()
+        }
+    }
+
+    companion object {
+        private const val ARG_PARAM = "param"
+
+        @JvmStatic
+        fun newInstance(param: TrendingMoviesParams) =
+            MoviesTrendingFragment().apply {
+                arguments = Bundle().apply {
```

Comment:
https://stackoverflow.com/questions/49342909/kotlin-extensions-for-android-how-to-use-bundleof